packages/babel-helper-validator-identifier/benchmark/util.mjs

```diff
@@ -1,13 +1,17 @@
 export function report(event) {
   const bench = event.target;
-  const factor = bench.hz < 100 ? 100 : 1;
   const timeMs = bench.stats.mean * 1000;
   const time =
     timeMs < 10
       ? `${Math.round(timeMs * 1000) / 1000}ms`
       : `${Math.round(timeMs)}ms`;
-  const msg = `${bench.name}: ${
-    Math.round(bench.hz * factor) / factor
-  } ops/sec ±${Math.round(bench.stats.rme * 100) / 100}% (${time})`;
+  const msg = `${bench.name}: ${formatNumber(bench.hz)} ops/sec ±${
+    Math.round(bench.stats.rme * 100) / 100
+  }% (${time})`;
   console.log(msg);
 }
+
+function formatNumber(x) {
+  if (x < 100) return String(Math.round(x * 100) / 100);
+  return String(Math.round(x)).replace(/\d(?=(?:\d{3})+$)/g, "$&\u066c");
```

Comment:
U+066C is an arabic thousand separator which may not be generally recognized in non-Arabic locale. If you would like the number here to be formatted according to your locale, consider use `Intl.NumberFormat` instead.

Reply:
I actually don't want to format the number according to locale, because 1) then everyone gets different output, and 2) in English locales thousands separators are commas, which I find super confusing — there are at least 3 ways to interpret a comma. I picked that character simply because there is no other explicitly for that purpose. Could've used U+00B4 or some other spacing/accent character. Doesn't really matter much, it's only to visually separate the digits so that you immediately see whether something's 3x faster or 3x slower.

Reply:
If we want to visually separate the digits, we can also go with `_`, the numeric separator in JS. I agree that it is more of a style issue and as long as we do it consistently, it is fine.

The `util.mjs` is also used in `packages/babel-parser` or elsewhere. Can you sync the changes here to other copies? Ideally we should make it a new package but we don't want to publish.

Reply:
Alright. Do you want me to add that to this PR or a separate one?

Reply:
I slightly prefer a separate PR and you can rebase this PR on benchmark util improvements.